rholang/src/main/bnfc/rholang_mercury.cf

```diff
@@ -110,13 +110,17 @@ GroundString.  Ground ::= String ;
 GroundUri.     Ground ::= Uri ;
 token Uri ('`' ((char - ["\\`"]) | ('\\' ["`\\"]))* '`') ;
 -- Collections:
-CollectList.   Collection ::= "[" [Proc] "]" ;
+CollectList.   Collection ::= "[" [Proc] Remainder "]" ;
 CollectTuple.  Collection ::= "(" [Proc] ")" ;
 CollectSet.    Collection ::= "Set" "(" [Proc] ")" ;
 CollectMap.    Collection ::= "{" [KeyValuePair] "}" ;
 KeyValuePairImpl.  KeyValuePair ::= Proc ":" Proc ;
 separator KeyValuePair "," ;
 
+-- Remainders:
+RemainderVar.   Remainder ::= "..." ProcVar ;
+RemainderEmpty. Remainder ::= "" ;
+
 token Var (((letter | '\'') (letter | digit | '_' | '\'')*)|(('_') (letter | digit | '_' | '\'')+)) ;
```

Comment:
You might want to consider calling this something else since it won't be a ProcVar for PInput/Contr.